packages/snjs/lib/services/history/history_manager.ts

```diff
@@ -74,11 +74,9 @@ export class SNHistoryManager extends PureService {
     this.persistable = await this.storageService!.getValue(
       StorageKey.SessionHistoryPersistable
     );
-    this.sessionHistory = await this.storageService!.getValue(
-      StorageKey.SessionHistoryRevisions
-    ).then((historyValue) => {
-      return SessionHistoryMap.FromJson(historyValue);
-    });
+    this.sessionHistory = SessionHistoryMap.FromJson(
+      this.storageService!.getValue(StorageKey.SessionHistoryRevisions)
```

Comment:
How does this work? getValue is async no?

Reply:
Ah nm, it's not async.

Reply:
Maybe getValue used to read something from disk? In any case it's actually completely sync (reads from memory) and I removed the modifier to address a lint warning. This refactor is in response to the compile error that followed